src/test/java/bio/overture/ego/controller/ApiKeyControllerTest.java

```diff
@@ -358,7 +358,10 @@ public void checkInvalidApiKey() {
     val response = initStringRequest().endpoint("o/check_api_key").body(params).post();
 
     val statusCode = response.getStatusCode();
-    assertEquals(HttpStatus.UNAUTHORIZED, statusCode);
+    /*assertEquals(HttpStatus.BAD_REQUEST, statusCode);
+    assertEquals("false", response.getStatusCode());*/
+      
+      assertEquals(HttpStatus.BAD_REQUEST, "false");*/
```

Comment:
We should be doing tests on the structure we expect to see:

statusCode = OK
body.valid = false